tensorflow/python/saved_model/load_test.py

```diff
@@ -47,6 +49,9 @@
 from tensorflow.python.framework import tensor_spec
 from tensorflow.python.framework import test_util
 from tensorflow.python.framework import versions
+from tensorflow.python.keras import Model
```

Comment:
tensorflow/python/keras is no longer maintained. Could you change the test so the objects inherit from the low-level model (tf.Module) instead?